lambdas/access_counts/pytest.ini

```diff
@@ -0,0 +1,9 @@
+[pytest]
+env =
```

Comment:
can we use smth like `[pytest.env]` to keep the entries unindented?

Reply:
I think ini lacks concept of (nested) tables so it would be just a section with dot in the name

Reply:
ok, soon (starting with pytest 9) we'll be able to [do that](https://docs.pytest.org/en/stable/reference/customize.html#pytest-toml)